modules/appeals_api/app/swagger/decision_reviews/v2/swagger.json

```diff
@@ -2490,7 +2490,6 @@
                                 "veteran": {
```

Comment:
The absence of changes to the API description at the top of this file indicate to me that code to modify the description isn't working as expected.